lib/tasks/import.rake

```diff
@@ -16,13 +15,12 @@ namespace :import do
     documents_to_import = WhitehallMigration::DocumentImport.where(whitehall_migration: whitehall_migration).count
     puts "Identified #{documents_to_import} documents to import"
 
-    puts whitehall_migration_url(whitehall_migration, host: Plek.new.external_url_for("content-publisher"))
+    puts whitehall_migration_url(whitehall_migration, host: Plek.new.external_url_for("content-publisher")).to_s
```

Comment:
I'm not sure this needs to_s whitehall_migration_url already returns a string. I couldn't see anything in the commit message regarding why this was needed.

Reply:
If memory serves correctly I think this was an automatic change by the linter, I believe it works without to_s